packages/components/src/tip/index.js

```diff
@@ -1,8 +1,17 @@
 /**
- * Internal dependencies
+ * WordPress dependencies
  */
-import { SVG, Path } from '../';
+import { SVG, Path } from '@wordpress/primitives';
```

Comment:
I changed this to make it possible to add types to Tip. Is there a reason why we weren't importing this directly from primitives?